GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/GameDataController.cs

```diff
@@ -0,0 +1,90 @@
+#define ONLINE
+//#define OFFLINE
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+using static Utils.Server;
+
+
+public class GameDataController
+{
+    private const string FILE_NAME = "data.json";
+    private static readonly string DATA_PATH = Application.persistentDataPath + "/" + FILE_NAME;
+
+    private static GameData _gameData;
+
+    public static void SaveGameData()
+    {
+#if OFFLINE
+        SaveGameDataOffline();
+#elif ONLINE
+        SaveGameDataOnline();
+#endif
+    }
+
+    private static void SaveGameDataOffline()
+    {
+        File.WriteAllText(DATA_PATH, JsonUtility.ToJson(_gameData, true));
+    }
+
+    private static void SaveGameDataOnline()
+    {
+        var values = new Dictionary<string, string>
+        {
+            ["gameData"] = JsonUtility.ToJson(_gameData)
+        };
+        sendUnityWebRequest(values, ServerURLs.SAVE_GAME_DATA_URL);
+    }
+
+    public static void LoadGameData()
+    {
+        Debug.Log(DATA_PATH);
+#if OFFLINE
+        LoadGameOffline();
+#elif ONLINE
+        LoadGameOnline();
+#endif
+    }
+
+    private static void LoadGameOnline()
+    {
+
+      ServerResponseModel serverResponse = sendUnityWebRequest(new Dictionary<string, string>(), ServerURLs.GET_GAME_DATA_URL);
+      if(serverResponse.success){
+        Debug.Log(serverResponse.result);
+        _gameData = JsonUtility.FromJson<GameData>(serverResponse.result);
+      }else{
```

Comment:
}
else
{

Reply:
Are you using JetBrains Rider? Use Rider -> Code -> Reformat Code

Reply:
Yep, I will do that.